grpcx/server_test.go

```diff
@@ -110,3 +110,71 @@ func TestWithRequestResponseLogs(t *testing.T) {
 		a.Contains(w.String(), fmt.Sprintf(`INFO gRPC Message FIELDS ctx_full_method=%s ctx_request_content={"UserID":"%s"} ctx_response_content={"Result":"%s"} ctx_response_error=%s`, method, userID, result, expectedErr.Error()))
 	})
 }
+
+func TestWithErrorLogs(t *testing.T) {
+	t.Parallel()
+	a := assert.New(t)
+
+	t.Run("logs nothing if no error", func(t *testing.T) {
+		w := bytes.NewBufferString("")
+		l := logx.New(logx.WriterOpt(w))
+
+		ctx := context.Background()
+		req := struct {
+			UserID string
+		}{
+			UserID: userID,
+		}
+
+		result := "ok"
+		expected := struct {
+			Result string
+		}{
+			Result: result,
+		}
+
+		info := &grpc.UnaryServerInfo{FullMethod: method}
+		handler := grpc.UnaryHandler(func(ctx context.Context, req interface{}) (interface{}, error) {
+			return expected, nil
+		})
+
+		interceptor := grpcx.WithErrorLogs(l)
+		resp, err := interceptor(ctx, req, info, handler)
+
+		a.NoError(err)
+		a.Equal(expected, resp)
+		a.Equal("", w.String())
+	})
+
+	t.Run("logs error", func(t *testing.T) {
+		w := bytes.NewBufferString("")
+		l := logx.New(logx.WriterOpt(w))
+
+		ctx := context.Background()
+		req := struct {
+			UserID string
+		}{
+			UserID: userID,
+		}
```

Comment:
you could move these repeated lines out of the func and share it between the two tests, to save some space.

Reply:
done :)